src/Compilers/Core/Portable/Symbols/ITupleTypeSymbol.cs

```diff
@@ -14,7 +13,7 @@ namespace Microsoft.CodeAnalysis
     /// </remarks>
     public interface ITupleTypeSymbol : ITypeSymbol
```

Comment:
@AlekseyTs Nevermind ;-) Reading the comments more carefully, I think you already clarified that new members would be better. I'll go that way.